packages/Algernon.package/ALGCore.class/instance/ifShouldHandleAsQuickAccess.do..st

```diff
@@ -0,0 +1,6 @@
+events-processing
+ifShouldHandleAsQuickAccess: anEvent do: aBlock
+    ^ ((anEvent modifierString sameAs: self class quickAccessModifier , ' ')
+            and: ['123456789' includes: anEvent keyCharacter])
+        ifTrue: [aBlock value]
+		ifFalse: [ nil ]
```

Comment:
```suggestion
    ((anEvent modifierString sameAs: self class quickAccessModifier , ' ')
            and: ['123456789' includes: anEvent keyCharacter])
        ifTrue: [aBlock value]
```
As the return value is never used, his should work as well.

Reply:
What?